fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/JobsServletSpec.groovy

```diff
@@ -171,4 +240,74 @@ class JobsServletSpec extends Specification {
         // Make the call
         httpCall.request().get(String.class)
     }
+
+    String getPaginatedExpectedResponse1(){
+        """
+        {
+          "rows": [
+            {
+              "dateTime": "2016-01-12 00:00:00.000",
+              "pageViews": 111
+            }
+          ],
+          "meta": {
+            "pagination": {
+              "last": "http://localhost:9998/jobs/ticket3p/results?asyncAfter=5&perPage=1&page=3",
+              "next": "http://localhost:9998/jobs/ticket3p/results?asyncAfter=5&perPage=1&page=2",
+              "currentPage": 1,
+              "rowsPerPage": 1,
+              "numberOfResults": 3
+            }
+          }
+        }
+       """
+    }
+
+    String getPaginatedExpectedResponse2(){
```

Comment:
I would suggest swapping the name of this method, with the name of `getPaginatedExpectedResponse3`. Basically, I think we should have the number in each method name line up with the page being returned.